spec/models/user_spec.rb

```diff
@@ -7,11 +7,11 @@
 RSpec.describe User do
   let!(:user) { create(:user) }
 
-  context 'validations' do
+  context 'when validations' do
```

Comment:
I guess with validations is a better option than when validations as mentioned in a comment above.